### PR TITLE
[WIP] Refactored

### PR DESCRIPTION
**Pull request by nil0bject**
moved stuff around
added stuff
not sure if i did the highlight stuff correctly..... eek
lulz, bad commit message, sorry

Hey @SorchaAbel
Don't merge it just yet. I'm not 100% on it. needs further review.

### CODE REVIEW

File: _posts/2015-11-15-sinatra-app.markdown [hunk 1]
```diff
@@ -11,79 +11,166 @@ permalink: sinatra-app-guide
...
+
+eg.
+```
+# My awesome Sinatra app!
+This app is like the RailsGirls Ideas app, but using Sinatra instead of Rails!
+```
+
+We will add more documentation as the project evolves.
+
+## *3.* Git
```
Comment (nil0bject) [inline _posts/2015-11-15-sinatra-app.markdown]: I thought that git should be an initial step and that commits should keep happening throughout the tutorial. I asked a previous RailsGirls student and she agreed it's important to learn the correct processes for development and practice them. I got a bit slack towards the end of the tutorial with reminding to commit. It does seem a bit un-DRY....

File: _posts/2015-11-15-sinatra-app.markdown [hunk 2]
```diff
@@ -11,79 +11,166 @@ permalink: sinatra-app-guide
...
+- [ ] Follow the instructions on GitHub under the heading `…or push an existing repository from the command line`
+
+They will be similar to below. DO NOT copy the lines below. You must push to your own newly created repo
+
+{% highlight sh %}
+git remote add origin https://github.com/YourName/project_name.git
+git push -u origin master
+{% endhighlight %}
 
-To install this gem, enter the following commands in the terminal window:
```
Comment (nil0bject) [inline _posts/2015-11-15-sinatra-app.markdown]: Wasn't sure why the sinatra gem was being installed globally, so i removed this part

File: _posts/2015-11-15-sinatra-app.markdown [hunk 3]
```diff
@@ -126,7 +213,7 @@ cd ..
 mkdir db
 {% endhighlight %}
 
-We are almost there. The final step in our skeleton structure for our application is to create a `config.ru` file in the root of the project. The `config.ru` file is a convention that is required by certain deployment procedures and tools (like shotgun, tux, and Heroku).
+We are almost there. The final step in our skeleton structure for our application is to create a `config.ru` file in the root of the project. The `config.ru` file is a convention that is required by certain deployment procedures and tools (like shotgun, tux, and Heroku). We also create a `Rakefile` to describe any automated tasks.
```
Comment (nil0bject) [inline _posts/2015-11-15-sinatra-app.markdown]: we're not actually using config.ru
if we do, we can change the command to start the server to `rackup`

File: _posts/2015-11-15-sinatra-app.markdown [hunk 4]
```diff
@@ -139,18 +226,40 @@ By the time our application is complete, it should look similar to the folder st
 
 <img src="/images/sinatra_visual_tree.png">
 
-## *5.* Setup Coding
+### *2b.* Commit the skeleton to git
```
Comment (nil0bject) [inline _posts/2015-11-15-sinatra-app.markdown]: not sure if this is overkill...
maybe an appendix?

File: _posts/2015-11-15-sinatra-app.markdown [hunk 5]
```diff
@@ -139,18 +226,40 @@ By the time our application is complete, it should look similar to the folder st
...
 We are now ready to begin developing our application.
 
+### *3b.* Sinatra
+
 The first step is to set up the `application.rb` file. Double click on the file to open it (you will find it under the `config` folder). Take some time to understand what we are about to put into this file.
-Note you must save all files in cloud9 (`cmd + s` or menu options `file -> save`)
+Note you must save all files in cloud9 (`command ⌘ + s`(mac), `control + s`(windows and linux) or menu options `File -> Save`)
 
 {% highlight sh %}
+# require gems and classes
```
Comment (nil0bject) [inline _posts/2015-11-15-sinatra-app.markdown]: the syntax is pretty cryptic in this file to keep it nice and short, so comments are kinda required for learning purposes

File: _posts/2015-11-15-sinatra-app.markdown [hunk 6]
```diff
@@ -226,10 +336,6 @@ class CreateIdeas < ActiveRecord::Migration
       t.timestamps null: false
     end
   end
-
```
Comment (nil0bject) [inline _posts/2015-11-15-sinatra-app.markdown]: the change method handles `up` and `down`

File: _posts/2015-11-15-sinatra-app.markdown [hunk 7]
```diff
@@ -671,32 +818,10 @@ ruby config/application.rb -p $PORT -o $IP
...
-git push -u origin master
-{% endhighlight %}
-
+Follow the heroku guide: [Pushing to Heroku](/heroku)
 
-## *15.* The End
+## *14.* Fin (The End)
 
-Every app should be accompanied by a README.md document. Edit this file with details about your app and push this update to Github (use the steps outlined above)
+Last bit of house keeping to do is to update the `README` and commit the final app to GitHub. Edit the `REAADME.md` with details about your app and push this update to GitHub (use the steps outlined above).
```
Comment (nil0bject) [inline _posts/2015-11-15-sinatra-app.markdown]: typo

**Comment by SorchaAbel** [comment]
Fantastic Nick :)

**Pull request merged**